src/pages/check-answers/template.njk

```diff
@@ -49,17 +49,17 @@
             },
             {
                 key: {
-                text: "Telling Civil Service Pension Scheme"
+                text: "Telling Other Service Pension Providers"
             },
                 value: {
-                text: mapping['tell-civil-service-pension'][session["tell-civil-service-pension"]]
+                text: mapping["other-pension-providers"]
             },
                 actions: {
                 items: [
                     {
-                        href: "/civil-service-pension",
+                        href: "/other-pension-providers",
                         text: "Change",
-                        visuallyHiddenText: "Allows the response to tell the civil service pension scheme to be changed"
+                        visuallyHiddenText: "Allows the other pension providers to be changed"
```

Comment:
This should follow the word "Change" to make a complete sentence when read out by a screen reader. Something like "which other pension providers to tell" ?